lgr/logger.go

```diff
@@ -1,24 +1,45 @@
 package lgr
 
 import (
-	"log"
+	"fmt"
 	"net/http"
 
 	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
 )
 
 // Logger represents logger.
 type Logger struct{ log *zap.SugaredLogger }
 
 // New initialize logger
-func New() *Logger {
-	// TODO: Extend config and make adjustments
-	logger, err := zap.NewDevelopment()
+func New(logLevel string) (*Logger, error) {
+	level, err := zapcore.ParseLevel(logLevel)
 	if err != nil {
-		log.Fatalf("Failed creating new logger: %v", err)
+		return nil, fmt.Errorf("error with logger level parsing: %w", err)
 	}
 
-	return &Logger{logger.Sugar()}
+	cfg := zap.Config{
+		Encoding:         "console",
+		Level:            zap.NewAtomicLevelAt(level),
+		OutputPaths:      []string{"stderr"},
+		ErrorOutputPaths: []string{"stderr"},
+		EncoderConfig: zapcore.EncoderConfig{
+			MessageKey: "message",
+
```

Comment:
Please remove empty lines like this one.

Reply:
it is already fixed